assets/base/i18n/sv.json

```diff
@@ -96,13 +96,39 @@
   "camera_type.video.option": "Spela in video",
   "center_panel.archived.closeChannel": "Stäng kanalen",
   "channel_add_members.add_members.button": "Lägg till medlemmar",
+  "channel_bookmark.add.detail_title": "Titel",
+  "channel_bookmark.add.emoji": "Lägg till emoij",
```

Comment:
Letters flipped in emoji.

Reply:
Sharp eye @phieri !
Fixed it through Weblate and pushed the changes